Add Surveys product page (#6580)
PostHog/posthog.com: contents/surveys/index.mdx

---
title: Surveys
subtitle: <span class="text-red font-semibold bg-yellow/10">Gauge product market fit</span>, measure Net Promoter Score (NPS), schedule user interviews and more
productHero: {
  video: ../images/tutorials/survey/surveys_hero_small.mp4
}
template: product
productFeatures: [
    {
        icon: 'Graph',
        title: 'Data visualization for results',
        description: 'Qualitative data? Quantative data? Just get the best of both.'
    },
    {

        icon: 'Bell',
        title: 'Slack notifications',

>         description: 'Because having to paste new responses in one-by-one is a PITA.'

Reviewer: I don't think non-British people will understand what PITA means sadly 😢 
```suggestion
        description: 'Because having to paste new responses in one-by-one is tedious.'
```

    },
    {
        icon: 'Clock',
        title: 'Customizable wait periods',
        description: 'Users hate too many pop-ups, and you do too. So do we.'

    },
]
productSections: [
  [
    {
        marquee: [

>         {text: "Do you have time to speak to our Product Manager?", url: "/tutorials/feedback-interviews-site-apps"},

Reviewer: ```suggestion
        {text: "Do you want to give your feedback in a user interview?", url: "/tutorials/feedback-interviews-site-apps"},
```

        {text: "How satisfied are you with the support you've received?",},
        {text: "How likely are you to recommend this product to a friend?", url: "/tutorials/nps-survey"},
        {text: "How would you feel if you could no longer use this product?",},
        {text: "How do I write good product survey questions?", url: "/blog/product-survey-questions"},
        {text: "Do you like pineapple on pizza?",},
      ]
    },
  ],
  [
    {
      image: ../images/products/surveys/question-types.png,
    },
    {
      align: 'center',
      title: "Ask whatever you want, however you want",
      sections: [
        {
          features: [
                      {
                        title: "Single or multiple choice questions",
                        description: "Let users select from a list of pre-set responses"
                      },
                      {
                        title: "Open text fields",
                        description: "Gather open-ended, unstructured feedback as free text"
                      },
                      {
                        title: "Number and emoji rating",

>                         description: "Gather NPS results, or just ask users if they're 😊 or 😊"

Reviewer: ```suggestion
                        description: "Gather NPS results, or just ask users if they're 😊 or 😢"
```

                      {
                        title: "Target by property",
                        description: "Focus on users with certain charateristics, such as early adopters"
                      },
                      {
                        title: "Target by feature flags",
                        description: "Survey users involved in betas or on new versions by linking to a flag"
                      },
                  ]
        },
      ]
    },
  ],
  [
    {
        image: ../images/products/surveys/mode.png
    },
    {
      align: 'center',
      title: "Launch popups quickly, or build your own",
      sections: [
        {
          features: [
                      {
                        description: "Use our customizable popup for a quick setup, and modify the colours to match your brand in seconds."
                      },
                      {
                        description: "Or, build your own custom UI and use our API to send results to PostHog for deeper analysis."
                      },
                  ]
        },
      ]
    },
  ]

]
productMainCTA: {
  title: Get started - free,
  url: https://app.posthog.com/signup,

>   subtitle: "Create your first surveys now, for free.",

Reviewer: ```suggestion
  subtitle: "Create your first survey now, for free.",
```
